bedrock/settings/base.py

```diff
@@ -596,93 +596,312 @@ def facebook_tab_url_lazy():
-    'es-ES': DONATE_SPANISH_LINK,
-    'es-MX': DONATE_SPANISH_LINK,
-    'fr': (
-        'https://donate.mozilla.org/fr/?presets=100,50,25,15&amount=50'
-        '&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org'
-        '&utm_medium=referral&utm_content={source}&currency=eur'
-    ),
-    'he': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-ils/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_ILS'
-    ),
-    'hu': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-huf/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_HUF'
-    ),
-    'ja': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-jpy/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_JPY'
-    ),
-    'nb-NO': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-nok/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_NOK'
-    ),
-    'nn-NO': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-nok/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_NOK'
-    ),
-    'pl': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-pln/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_PLN'
-    ),
-    'pt-BR': (
-        'https://donate.mozilla.org/pt-BR/?presets=375,187,90,55&amount=187'
-        '&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org'
-        '&utm_medium=referral&utm_content={source}&currency=brl'
-    ),
-    'ru': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-rub/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_RUB'
-    ),
-    'sv-SE': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-sek/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_SEK'
-    ),
-    'th': (
-        'https://sendto.mozilla.org/page/content/paypal-donate-thb/?'
-        'source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content=PPcurrency_THB'
-    ),
+DONATE_LINK = ('https://donate.mozilla.org/{locale}/?presets={presets}'
+    '&amount={default}&ref=EOYFR2015&utm_campaign=EOYFR2015'
+    '&utm_source=mozilla.org&utm_medium=referral&utm_content={source}'
+    '&currency={currency}')
+
+DONATE_PARAMS = {
+    'default': {
```

Comment:
This is mostly a nit, but I'd find it easier to understand and more explicit if this entry were `en-US` instead of `default`, then have the default for a locale that's not in the list be `en-US`. That way the code is clear that we're defaulting to en-US. Also it means that the majority of the traffic will go down the main code path (finding the locale in the dict) instead of the alternate branch (using the default).